clients/drcachesim/tests/burst_gencode.cpp

```diff
@@ -132,6 +132,12 @@ class code_generator_t {
         // instead of the pop into pc which assumes the entry pushed lr?
         instrlist_append(ilist, INSTR_CREATE_bx(dc, opnd_create_reg(DR_REG_LR)));
 #else
+#    ifdef X86
+        instrlist_append(ilist,
```

Comment:
I would add a comment explaining that we're inserting this zero-iter rep string loop to test emulation-marked code written to the encoding file.